web/src/styles/components/card.scss

```diff
@@ -28,3 +28,11 @@
   font-size: 0.88rem;
   font-weight: 600;
 }
+
+.usa-prose > h3,
+.usa-prose > .h3-styling,
+.h3-styling,
+h3,
+.support-action-card {
```

Comment:
pebble - all of the typography has been set. I'd confirm with Design about the line height change. I don't see any updates in the design system. I think it might be an oversight. If there is a global change to the H3 styling, I'd update the line height in typography.scss. Right now you are overriding the line height for all H3s, and it's noted in card.scss

Reply:
So I merged, without re-freshing my page so I didn't see these comments until after. 

but you're totally right that this applies to wayyy more than I wanted it to. (I got my CSS selectors mixed up).

That being said its not really a change to design actually becuase now we're using line height to make something happen that was previously 2 elements with padding. Lmk if that doesn't sense and I can clarify what I mean better. 

<img width="357" alt="image" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/0d9c931b-081c-450d-b45f-6b26ea1a9cb8">
vs
<img width="390" alt="image" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/854d1745-3a04-4d38-9b22-165dbc1891e5">